Automated Smoketest: :x:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 8ececf25cb70b0402ab4f2bbd31be9df8c1b0758
Number of Error(s): 5
/mnt/disk1/home/vamosax/smoketest/PR-12356/HPCC-Platform/system/jlib/jutil.cpp:2283:79: error: \xe2\x80\x98visibility\xe2\x80\x99 attribute ignored on non-class types [-Werror=attributes]
cc1plus: all warnings being treated as errors
make[2]: *** [system/jlib/CMakeFiles/jlib.dir/jutil.cpp.o] Error 1
make[1]: *** [system/jlib/CMakeFiles/jlib.dir/all] Error 2
make: *** [all] Error 2

[Build error:](https://gist.github.com/4d89dd46f3325135503a25206085f568)
Build log [Build log](https://gist.github.com/HPCCSmoketest/4d89dd46f3325135503a25206085f568#file-result-19-03-26-07-59-56-log)

--- a/plugins/javaembed/javaembed.cpp
+++ b/plugins/javaembed/javaembed.cpp
@@ -4329,6 +4329,9 @@ protected:
         ret.append(classname).append('.');
         switch (persistMode)
         {
+        case persistThread:
+            ret.append(__uint64(GetCurrentThreadId()));
+            break;
         case persistGlobal:
             ret.append("global");
             break;

